core/sr-primitives/src/weights.rs

```diff
@@ -74,3 +81,59 @@ impl Default for TransactionWeight {
 		TransactionWeight::Basic(0, 1)
 	}
 }
+
+/// Representation of a fee multiplier.
+///
+/// This is basically a wrapper for the `Fixed64` type a slightly tailored multiplication to u32
+/// in the form of the `apply_to` method.
+#[cfg_attr(feature = "std", derive(Debug, Copy, Clone))]
+#[derive(Encode, Decode, Default, PartialEq, Eq, PartialOrd, Ord)]
+pub struct FeeMultiplier(Fixed64);
```

Comment:
I should point out to anyone who might argue that this is not needed and it just wraps a number:

- That I agree that it looks like that.
- But I really prefer keeping the `apply_to` method to keep the code in `executive` concise and expressive.
  - And what `apply_to` is doing is neither a general function for `Fixed64` nor a `add` or `saturating_add` operation.

Hence settled for this.